### ISSUE TITLE
Having multiple surveys generates invalid html (duplicate id='current_user_id')

### ISSUE THREAD

**Issue by jakewendt** [NUARIG/surveyor#48]
<div id='survey_list'>
    <ul>
      <li>
        <form action="/surveys/home_exposure_survey_2" method="post"><div style="margin:0;padding:0;display:inline"><input name="authenticity_token" type="hidden" value="0z2orvAjNy5gXt3DzDHQcbwwhiVBO958pfpTcdmkL5o=" /></div>
          <input id="current_user_id" name="current_user_id" type="hidden" value="1" />
          Home Exposure survey
          &nbsp;
          <input name="commit" type="submit" value="Take it" />
        </form>
      </li>

```
  <li>
    <form action="/surveys/home_exposure_survey" method="post"><div style="margin:0;padding:0;display:inline"><input name="authenticity_token" type="hidden" value="0z2orvAjNy5gXt3DzDHQcbwwhiVBO958pfpTcdmkL5o=" /></div>
      <input id="current_user_id" name="current_user_id" type="hidden" value="1" />
      Home Exposure survey
      &nbsp;
      <input name="commit" type="submit" value="Take it" />
    </form>
  </li>
</ul>
```

</div>

The id isn't needed so should probably be set to 'nil' in `app/views/surveyor/new.html.haml` to avoid this.

<pre>
= hidden_field_tag("current_user_id", @current_user.nil? ? nil : @current_user.id, :id => nil)
</pre>

**Comment by brian-lc** [comment]
Do you mean multiple surveys on the same page? We really didn't design the plugin to support this but I'll take a look at the issue.

**Comment by jakewendt** [comment]
Yes.  If you create multiple surveys and then go to "/surveys", the presence of two tags with id "current_user_id" is invalid.

I've included the fix and its shouldn't be invasive, so it should be easy to include in the next version if desired.

**Comment by yoon** [comment]
current user id not needed in the view, set in SurveyorController. closed by a097ff4580c8f6c3223e32a308ab9c8f54572251. thanks jakewendt
